src/index.js

```diff
@@ -177,48 +180,101 @@ class Saturn {
   }
 
   async _reportLogs () {
-    if (this.logs.length) {
-      await fetch(
-        'https://twb3qukm2i654i3tnvx36char40aymqq.lambda-url.us-west-2.on.aws/',
-        {
-          method: 'POST',
-          body: JSON.stringify({
-            bandwidthLogs: this.logs
-          })
-        }
-      )
-      this.logs = []
+    if (!this.logs.length) {
+      return
+    }
+
+    let bandwidthLogs = this.logs
+    if (this.hasPerformanceAPI) {
+      bandwidthLogs = this._matchLogsWithPerformanceMetrics(bandwidthLogs)
     }
```

Comment:
```suggestion
    const bandwidthLogs = this.hasPerformanceAPI
      ? this._matchLogsWithPerformanceMetrics(this.logs)
      : this.logs
```

nit: this way you can use a safer `const`